### ISSUE TITLE
Not working with paper?

### ISSUE THREAD

**Issue by dly2424** [Macuyiko/minecraft-python#57]
From what I understand, this does not work with paper, and perhaps spigot? Just bukkit?

I see you're referencing the Bukkit API and on line 38 of your current version as of writing this you have:

`WORLD  = SERVER.getWorlds().get(0)`

But with paper this seems to return `None` as it is using 

`SERVER = Bukkit.getServer()`

From line 37. I'm not familiar with how these server jars are structured, correct me if I'm wrong. Would there be a paper/spigot API that I could easily change?

**Comment by dly2424** [comment]
Seems like Paper supports Bukkit API fully, more or less. For whatever reason just doesn't work with 1.18.2.

Either they changed something in the Bukkit API or there's some dependency for the specific version the server is run on. 

TLDR: I ran this on 1.18.2 (latest at the time of writing) and it didn't work. Tried on spigot, didn't work. Only until I dropped my version down to 1.17.1.

**Comment by Macuyiko** [comment]
Indeed, both Spigot and Paper are Bukkit-compatible. The current release is compiled against Spigot 1.17, so you probably need to compile the plugin first with 1.18.2 for it to work (or wait until I can put up a new release).
